packages/firestore/test/integration/api/validation.test.ts

```diff
@@ -437,7 +437,7 @@ apiDescribe('Validation:', persistence => {
         return expectSetToFail(
           db,
           { foo: firebase.firestore.FieldValue.delete() },
-          'FieldValue.delete() can only be used with update() and set() with {merge:true} (found in field foo)'
+          'FieldValue.delete() cannot be used with set() unless you pass {merge:true}'
```

Comment:
Why did the field name disappear?

Reply:
Uh, turns out it didn't...  But I accidentally removed it from the test and chai's .to.throw() expectation just verifies that the error _contains_ the specified string, so it still passed.

We should probably look into making the expectations more strict, but I think we'd need to switch to using a regex and we'd need to do escaping and stuff, so it's a bit of a pain.  _contains_ is probably good enough for now.  

Fixed the test in any case.